src/pages/sites-and-applications.tsx

```diff
@@ -135,25 +131,26 @@ const SitesAndApplications = ({
 
     handleAddCollections({
       variables: addCollectionsInput(collectionObjs),
-      refetchQueries: [`getMySpace`],
+      refetchQueries: [`getUser`],
     })
     setSelectMode(false)
     setSelectedCollections([])
     router.push('/')
   }
 
   const handleAddToCollection = (
-    bookmark: BookmarkRecord,
+    bookmark: CMSBookmark,
     collectionId?: ObjectId
   ) => {
     if (collectionId) {
       handleAddBookmark({
         variables: {
+          url: bookmark.url,
+          label: bookmark.label,
           collectionId,
           cmsId: bookmark.id,
-          ...bookmark,
         },
```

Comment:
@jcbcapps this is one of the areas that was causing the test to fail, turned out after the fix of the types we were sending too much information to the bookmark. Thanks to @abbyoung for finding it.